tests/test_server.py

```diff
@@ -206,6 +206,7 @@ def test_srv_keep_alive(srv):
     assert not srv._keep_alive
 
 
+@pytest.mark.timeout(0)
```

Comment:
This test timed out, (I guess because of the addition of pytest-timeout) and modified it to fix it